src/screens/LoginScreen.tsx

```diff
@@ -33,11 +49,45 @@ export type LoginScreenProps = NativeStackScreenProps<RootStackParams, 'login'>;
 export const LoginScreen = (props: LoginScreenProps) => {
   const dispatch = useAppDispatch();
   const headerHeight = useHeaderHeight();
+  const formRef = useRef<FormikProps<any>>(null);
 
-  const { localUsernames, lastUsername } = useAppSelector(
+  const biometricUseStatus = useAppSelector(selectBiometricUseStatus);
+  const { localUsernames, lastUsername, signInStatus } = useAppSelector(
     state => state.account,
   );
 
+  useEffect(() => {
+    (async () => {
+      if (signInStatus === SignInStatus.INITIAL) {
+        const isBiometricSupport = await isBiometricSupported();
+
+        if (
+          isBiometricSupport &&
+          lastUsername &&
+          biometricUseStatus?.[lastUsername]
+        ) {
+          await LocalAuthentication.authenticateAsync({
+            promptMessage:
+              'To use Telios, you must have identity verification.',
```

Comment:
I suggest message prompt "To use Telios, identity verification required" @hexadecible, @Ellobo16